compiled_starters/c/app/main.c

```diff
@@ -0,0 +1,15 @@
+#include <stdio.h>
+
+int main() {
+  // You can use print statements as follows for debugging, they'll be visible when running tests
+  printf("Logs from your program will appear here!\n");
+
+  // Uncomment this block to pass the first stage
+  //   printf("$ ");
```

Comment:
Added a note below in the diff, the way we've indented commented code here causes incorrect indentation in the diff. Follow what other challenges do: 

<img width="719" alt="Screenshot 2024-06-05 at 11 48 42" src="https://github.com/codecrafters-io/build-your-own-shell/assets/3893573/2ebfcef2-2da2-4a51-bdfa-e00c9dc1dcd7">